dcp_prototype/backend/browser/Makefile

```diff
@@ -0,0 +1,26 @@
+SHELL=/bin/bash -o pipefail
+tests:=$(wildcard tests/test*.py)
+
+deploy:
+	$(MAKE) deploy -C api
+
+destroy:
```

Comment:
I wonder if we need a `destroy` target considering how infrequently/undesirable we'd want to run that. Also given how straightforward the command is, `chalice delete --stage <stage>`, if we did intend to run it.

Reply:
this may not be the only thing that needs destroying in the future. This set a presidence on how we go about setting up and cleaning up multiple deployed resources.